gettsim/transfers/rente.py

```diff
@@ -319,7 +319,9 @@ def zugangsfaktor(
     return out
 
 
-def regelaltersgrenze(geburtsjahr: IntSeries) -> FloatSeries:
+def regelaltersgrenze(
+    geburtsjahr: IntSeries, ges_renten_vers_params: dict
+) -> FloatSeries:
     """Calculates the age, at which a worker is eligible to claim his full pension.
 
     Parameters
```

Comment:
Please add `ges_renten_vers_params` also to the Parameters description in the docstring.